frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -40,6 +40,7 @@ import { insightDataLogic } from 'scenes/insights/insightDataLogic'
 import { queryNodeToFilter } from '~/queries/nodes/InsightQuery/utils/queryNodeToFilter'
 import { InsightVizNode } from '~/queries/schema'
 import { groupsModel } from '~/models/groupsModel'
+import { Tag } from 'antd'
```

Comment:
Should we be using a `LemonTag` here?

Reply:
Generally it'd be good to remove all of the antd Rows, Cols and stuff at the same time. They never get removed if we don't remove them 🙈

Reply:
![image](https://github.com/PostHog/posthog/assets/7115141/057eb940-19db-436b-bfbc-42e1e6e3ad79)

Reply:
well, I got rid of this problem by not importing Tag here 😅